app/(gcforms)/[locale]/(form administration)/form-builder/components/shared/right-panel/treeview/handlers/handleOnDrop.ts

```diff
@@ -204,6 +204,8 @@ export const handleOnDrop = async (
       newGroups[String(originParent?.index)] = {
         name: String(originParent?.data.titleEn),
         elements: originGroupElements,
+        titleEn: originParent?.data.titleEn,
+        titleFr: originParent?.data.titleFr,
```

Comment:
I was seeing an error about these being missing from the Group type. This was a best guess with originalParent and targetParent.

Reply:
@dsamojlenko 

Can you validate this?